packages/common/src/service.js

```diff
@@ -51,14 +51,13 @@ function Service(config, options) {
   });
 
   this.makeAuthenticatedRequest = util.makeAuthenticatedRequestFactory(reqCfg);
-
   this.authClient = this.makeAuthenticatedRequest.authClient;
   this.baseUrl = config.baseUrl;
   this.getCredentials = this.makeAuthenticatedRequest.getCredentials;
   this.globalInterceptors = arrify(options.interceptors_);
   this.interceptors = [];
   this.packageJson = config.packageJson;
-  this.projectId = options.projectId;
+  this.projectId = options.projectId || '{{projectId}}';
```

Comment:
Got it, so then my next question would be if `grpc` services offer `interceptors` and if so, could we not just create a globalInterceptor at the `Service` level that would replace the `{{projectId}}`?